Languages/Dutch/Keyed/Multiplayer.xml

```diff
@@ -0,0 +1,130 @@
+  <MpChangeUsernameInfo>U kunt uw gebruikersnaam wijzigen in de wijzigingsinstellingen.</MpChangeUsernameInfo>
+  
+  <MpPlayerConnected>{0} heeft zich aangesloten</MpPlayerConnected>
+  <MpPlayerDisconnected>{0} is vertrokken</MpPlayerDisconnected>
+  
+  <MpReplayOutdated>(Outdated)</MpReplayOutdated>
+  <MpReplayOutdatedDesc1>Deze herhaling is gemaakt voor de protocolversie {0}, maar je bent bezig met de versie die je gebruikt... {1}.</MpReplayOutdatedDesc1>
+  <MpReplayOutdatedDesc2>De herhalingen worden mogelijk niet correct afgespeeld.</MpReplayOutdatedDesc2>
+  <MpReplayOutdatedDesc3>Mogelijke incompatibiliteiten worden vermeld in het releaseverleden.</MpReplayOutdatedDesc3>
+  <MpReplayOutdatedDesc4>Dit bestand bevat aanvullende informatie en de hosting ervan zou goed moeten werken.</MpReplayOutdatedDesc4>
+  
+  <MpServerCloseConfirmation>Weet u zeker dat u de server wilt sluiten? Onbewaarde vooruitgang zal verloren gaan.</MpServerCloseConfirmation>
+  
+  <MpHostServer>Host een server</MpHostServer>
+  <MpSaveReplay>Opslaan van herhaling</MpSaveReplay>
+  
+  <MpConvert>Omzetten naar singleplayer</MpConvert>
+  <MpConverting>Aan het omzetten</MpConverting>
+  
+  <MpUsername>Gebruikersnaam</MpUsername>
+  <MpShowPlayerCursors>Toon cursors van spelers</MpShowPlayerCursors>
+  <MpAutoAcceptSteam>Auto-accepteren Steam</MpAutoAcceptSteam>
+  <MpAutoAcceptSteamDesc>Accepteer automatisch alle inkomende aanvragen voor een stoomverbinding.</MpAutoAcceptSteamDesc>
+  <MpTransparentChat>Transparante chat</MpTransparentChat>
+  
+  <MpAutosaveSlots>Autosave Slots</MpAutosaveSlots>
+  <MpAutosaveSlotsDesc>Hoeveel autosave-bestanden te bewaren.</MpAutosaveSlotsDesc>
+  
+  <MpTraderLeavesIn>Handelaar vertrekt in {0}</MpTraderLeavesIn>
+  
+  <MpChatButton>Chat</MpChatButton>
+  <MpTradingButton>Handel</MpTradingButton>
+  
+  <MpChatPlayers>Spelers ({0}):</MpChatPlayers>
+  <MpSend>Verzenden</MpSend>
+  <MpSteamAcceptTitle>Steam-spelers accepteren</MpSteamAcceptTitle>
+  <MpSteamAcceptDesc>Klik om te accepteren</MpSteamAcceptDesc>
+  <MpSteamAccepted>Speler is geaccepteerd</MpSteamAccepted>
+  
+  <MpConnectingTo>Verbonden maken met {0}:{1}</MpConnectingTo>
+  <MpConnected>Aangesloten.</MpConnected>
+  <MpDownloading>Downloaden ({0}%)</MpDownloading>
+  
+  <MpSteamConnectingTo>Een verbinding met {0} aan het maken</MpSteamConnectingTo>
+  <MpSteamConnectingWaiting>We wachten op een bevestiging van de host</MpSteamConnectingWaiting>
+  
+  <MpTicksBehind>U bent {0} ticks vroeger.</MpTicksBehind>
+  <MpLowerGameSpeed>Overweeg de spelsnelheid te verlagen.</MpLowerGameSpeed>
+  <MpForceCatchUp>Klik om achterstand inhalen te forceren.</MpForceCatchUp>
+  
+  <MpDesynced>De staat van het spel is niet (meer) gesynchroniseerd.</MpDesynced>
+  <MpTryResync>Probeer hersynchronisatie</MpTryResync>
+  
+  <MpSaveReplayAs>Sla replay op als</MpSaveReplayAs>
+  <MpWillOverwrite>Dit zal overschrijven</MpWillOverwrite>

```

Comment:
This is iffy, what is the English translation?

Reply:
This will overwrite "..."/Dit zal "..." overschrijven

Reply:
"Dit zal overschreven worden: "